blocks.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"context"
+	"notes-service/grpc/notespb"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/emptypb"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+var _ notespb.NotesServiceServer = &notesService{}
+
+func (srv *notesService) AddBlock(ctx context.Context, in *notespb.AddBlockRequest) (*emptypb.Empty, error) {
+
```

Comment:
Is there a specific reason why all your functions start with an empty line? I would suggest removing it.